consensus/propagation/reactor.go

```diff
@@ -144,12 +144,6 @@ func (blockProp *Reactor) OnStart() error {
 
 func (blockProp *Reactor) OnStop() {
 	blockProp.cancel()
-	for _, peer := range blockProp.getPeers() {
```

Comment:
no need to manually close the channels, they will be garbage collected automatically once all the routines exit.